### PR TITLE
Make install button easier

### PR DESCRIPTION
**Pull request by Zexbe**
Fixes: https://github.com/rust-lang/www.rust-lang.org/issues/969
Possibly Fixes: https://github.com/rust-lang/www.rust-lang.org/issues/954
Possibly Fixes: https://github.com/rust-lang/www.rust-lang.org/issues/733

### **Before**
Before

### **After**
mini
small
larger

### CODE REVIEW

**Comment by pietroalbini** [comment]
Thanks for the PR!

As far as I'm aware this would be the only page with an 80% width paragraph, which is not consistent with the rest of the site. An alternative approach would be to keep the 100% width for the paragraph while having two buttons instead of one in the row below:

* [Download rustup-init.exe (32-bit)](https://static.rust-lang.org/rustup/dist/i686-pc-windows-msvc/rustup-init.exe)
* [Download rustup-init.exe (64-bit)](https://static.rust-lang.org/rustup/dist/x86_64-pc-windows-msvc/rustup-init.exe)

This would also provide yet another band-aid for https://github.com/rust-lang/rustup/issues/2179 (cc @kinnison).

Also, do not update the translations by changing the ftl files directly in the repo: the English language is the only one supposed to change, the other ones need to be updated through Pontoon.

**Comment by Zexbe** [comment]
Okay, ready to be reviewed

**Comment by pietroalbini** [comment]
This looks way better! Thanks!

Just a last small change, could you add some margin between the buttons and the following title?

**Comment by Zexbe** [comment]
Okay

File: locales/en-US/tools.ftl [hunk 1]
```diff
@@ -149,8 +153,8 @@ install-other-methods-link = Learn more
 
 ## components/tools/rustup.hbs
 
-tools-rustup-unixy = It looks like you’re running macOS, Linux, or another Unix-like OS. To download Rustup and install Rust, run the following in your terminal, then follow the on-screen instructions.
-tools-rustup-windows = It looks like you’re running Windows. To install Rust, download and run the following, and then follow the onscreen instructions.
+tools-rustup-unixy = It looks like you’re running macOS, Linux, or another Unix-like OS. To download Rustup and install Rust, run the following in your terminal, then follow the on-screen instructions. If you are using Windows, <a href="https://forge.rust-lang.org/infra/other-installation-methods.html">click here</a>
+tools-rustup-windows = It looks like you’re running Windows. To start using Rust, download the installer, then run the program and follow the onscreen instructions. If you are not using Windows, <a href="https://forge.rust-lang.org/infra/other-installation-methods.html">click here</a>
```
Comment (pietroalbini) [inline locales/en-US/tools.ftl]:
```suggestion
tools-rustup-windows = It looks like you’re running Windows. To start using Rust, download the installer, then run the program and follow the onscreen instructions. If you are not using Windows, <a href="https://forge.rust-lang.org/infra/other-installation-methods.html">click here</a>.
```

File: locales/en-US/tools.ftl [hunk 2]
```diff
@@ -149,8 +153,8 @@ install-other-methods-link = Learn more
 
 ## components/tools/rustup.hbs
 
-tools-rustup-unixy = It looks like you’re running macOS, Linux, or another Unix-like OS. To download Rustup and install Rust, run the following in your terminal, then follow the on-screen instructions.
-tools-rustup-windows = It looks like you’re running Windows. To install Rust, download and run the following, and then follow the onscreen instructions.
+tools-rustup-unixy = It looks like you’re running macOS, Linux, or another Unix-like OS. To download Rustup and install Rust, run the following in your terminal, then follow the on-screen instructions. If you are using Windows, <a href="https://forge.rust-lang.org/infra/other-installation-methods.html">click here</a>
```
Comment (pietroalbini) [inline locales/en-US/tools.ftl]:
```suggestion
tools-rustup-unixy = It looks like you’re running macOS, Linux, or another Unix-like OS. To download Rustup and install Rust, run the following in your terminal, then follow the on-screen instructions. If you are using Windows, <a href="https://forge.rust-lang.org/infra/other-installation-methods.html">click here</a>.
```

Comment (XAMPPRocky) [inline locales/en-US/tools.ftl]: Don't use "click here" for links, it's bad content. The link text should describe what you're clicking on.
```suggestion
tools-rustup-unixy = It looks like you’re running macOS, Linux, or another Unix-like OS. To download Rustup and install Rust, run the following in your terminal, then follow the on-screen instructions. See <a href="https://forge.rust-lang.org/infra/other-installation-methods.html">"Other Installation Methods"</a> if you are on Windows.
```

File: locales/en-US/tools.ftl (hunk 1 above)
Comment (XAMPPRocky) [inline locales/en-US/tools.ftl]: Same comment as above.

**Comment by Zexbe** [comment]
Okay, updated the text

File: locales/en-US/tools.ftl [hunk 3]
```diff
@@ -60,6 +60,10 @@ tools-install-page-title = Install Rust
 
 install-using-rustup-heading = Using rustup (Recommended)
 
+install-rustup32-button = Download <nobr>rustup-init.exe</nobr> <nobr>(32-bit)</nobr>
```
Comment (XAMPPRocky) [inline locales/en-US/tools.ftl]: `nobr` is [non-standard](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr). 
```suggestion
install-rustup32-button = Download <span class="nowrap">rustup-init.exe</span> <span class="nowrap">(32-bit)</span>
```

File: locales/en-US/tools.ftl [hunk 4]
```diff
@@ -60,6 +60,10 @@ tools-install-page-title = Install Rust
...
+
+install-rustup64-button = Download <nobr>rustup-init.exe</nobr> <nobr>(64-bit)</nobr>
```
Comment (XAMPPRocky) [inline locales/en-US/tools.ftl]:
```suggestion
install-rustup64-button = Download <span class="nowrap">rustup-init.exe</span> <span class="nowrap">(64-bit)</span>
```

**Comment by XAMPPRocky** [comment]
Okay, this looks looks good to me! Thank you again for your PR!

**Pull request merged**